src/javascript/_common/base/client_base.js

```diff
@@ -316,7 +316,7 @@ const ClientBase = (() => {
 
     const canChangeCurrency = (statement, mt5_login_list, is_current = true) => {
         const currency             = get('currency');
-        const has_no_mt5           = mt5_login_list.length === 0;
+        const has_no_mt5           = mt5_login_list ? mt5_login_list.length === 0 : true;
```

Comment:
can we change this to:
```suggestion
        const has_no_mt5           = !mt5_login_list || !mt5_login_list.length;
```
I think it's less confusing

Reply:
Sure. Will do.